samples/server/petstore/erlang-server/src/openapi_api.erl

```diff
@@ -475,7 +475,7 @@ validate_response('DeleteOrder', 404, Body, ValidatorState) ->
     validate_response_body('', '', Body, ValidatorState);
 
 validate_response('GetInventory', 200, Body, ValidatorState) ->
-    validate_response_body('map', 'integer', Body, ValidatorState);
+    validate_response_body('map', 'map', Body, ValidatorState);
```

Comment:
baseType has changed for this generator, is this correct?

Reply:
Response schema is:
```
              schema:
                type: object
                additionalProperties:
                  type: integer
                  format: int32
```
So it should be a map where the values bust all be of type integer.

Reply:
@tsloughter can you give this a look?